metrics-tracing-context/Cargo.toml

```diff
@@ -32,6 +32,7 @@ itoa = { version = "1", default-features = false }
 metrics = { version = "^0.21", path = "../metrics" }
 metrics-util = { version = "^0.15", path = "../metrics-util" }
 lockfree-object-pool = { version = "0.1.3", default-features = false }
+indexmap = "2.1"
```

Comment:
Can you add `default-features = false` here? It's fine if we need to re-enable a specific feature, but we don't want to include default features automatically.